src/css/_layout.scss

```diff
@@ -92,32 +92,100 @@
 	display: block;
 }
 
-.minibox1{
-	background-color:#052a3d;
-	width:70%;
-	display:block;
-	margin-left:auto;
-	margin-right:auto;
-  }
+.minibox1 {
+	background-color: #052a3d;
+	width: 70%;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
 
-  .spot1{
-	font-family: raleway;
-	  font-size: 20px;
-	  color: #f8f9eb;
-	  text-align: center;
+.spot1 {
+	font-family: raleway, sans-serif;
```

Comment:
Double check this is loaded with Google Fonts or a similar service - you can check using your phone (you'll need to run `npm run deploy` first like we talked about in class to be able to do this). If the fonts don't show on your phone, you'll know this isn't loaded correctly. It's easy to miss on your computer because you probably have it installed on your machine from working on the design.